javalib/src/main/scala/java/io/DataInputStream.scala

```diff
@@ -169,8 +185,15 @@ class DataInputStream(in: InputStream)
 
     def badFormat(msg: String) = throw new UTFDataFormatException(msg)
 
+    def getUnsignedByte(): Int =
+    try {
+    in.readUnsignedByte()
+    } catch {
+      case _: EOFException => -1
+    }
```

Comment:
This portion is misindented.

Reply:
Fixed, as is the underlying defect which caused scalafmt to accept the wrong indentation.